app/src/main/java/fi/sabriina/urbanhuikka/roomdb/Converters.kt

```diff
@@ -0,0 +1,25 @@
+package fi.sabriina.urbanhuikka.roomdb
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import fi.sabriina.urbanhuikka.card.Card
+
+
+class Converters {
+
+    @TypeConverter
+    fun fromString(value: String): Card? {
+        val gson = Gson()
+        if (value != null) {
```

Comment:
value is never null as stated in the function declaration. Is this check unnecessary or should the parameter be a nullable string?

Reply:
It's redundant. FromJson method will return null when needed.